src/components/AppContent.js

```diff
@@ -1,13 +1,98 @@
 import '../css/App.css';
+import React, { useState } from 'react';
 import KevContainer from './KevContainer.js';
 import QuoteContainer from './QuoteContainer.js';
+import StartButton from './StartButton.js';
+import ItsKevButton from './ItsKevButton.js';
+import NotKevButton from './NotKevButton.js';
+import Points from './Points.js';
+import ResetButton from './ResetButton.js';
+import newQuote from '../utils/quoteGenerator';
+
+const placeholderQuote = {author: "Placeholder", quotation: "Hit the button to start."};
+const endQuote = {author: "Placeholder", quotation: "That's all folks!"};
+let usedQuotes = [];
 
 const AppContent = () => {
+  const [quote, setQuote] = useState(placeholderQuote);
+  const [status, setStatus] = useState("start");
+  const [points, setPoints] = useState(0);
+
+  if (status === "reset") {
+    setQuote(placeholderQuote);
+    setPoints(0);
+    setStatus("open");
+  }
+
+  if (status === "end") {
+    document.getElementById("its-kev-button").setAttribute("style", "display: none;");
+    document.getElementById("not-kev-button").setAttribute("style", "display: none;");
+  }
+
+  const handleStartClick = (event) => {
+    if (usedQuotes.length === 13) {
+      setQuote(endQuote); 
+      setStatus("end");
+      } else {
+          let newQuoteObject;
+          do {
+            newQuoteObject = newQuote();
+          } while (usedQuotes.includes(newQuoteObject.id));
+          usedQuotes.push(newQuoteObject.id);
+          console.log(usedQuotes);
+          setQuote(newQuoteObject);
+          setStatus("open");
+        }
+  }
+
+  const handleResetClick = () => {
+    if (status === "end") {
+      document.getElementById("its-kev-button").setAttribute("style", "display: inital;");
+      document.getElementById("not-kev-button").setAttribute("style", "display: initial;");
+    }
+    setStatus("reset");
+    usedQuotes = [];
+  }
+
+  const handleKevClick = () =>  { 
+    if (quote.author === "Kevin McCloud") { 
+          setStatus("correct-kev");
+          setPoints(points => {
+              return points + 1;
+          });
+    } else {
+          setStatus("incorrect-kev");
+        }
+  }
+
+  const handleNotKevClick = () =>  { 
+    if (quote.author !== "Kevin McCloud") { 
+        setStatus("correct-notkev");
+        setPoints(points => {
```

Comment:
If you are doing an instant return, you can get away with the following (which I think looks nicer too):
`setPoints(prevPoints => prevPoints + 1);`